routes/web.php

```diff
@@ -11,24 +11,35 @@
 */
 
 
-Route::get('/', function () {
-    return view('welcome');
-})->middleware('guest');
+// Route::get('/', function () {
+//     return view('welcome');
+// })->middleware('guest');
+
+
+Route::get('/','HomeController@show' );
 
 // remove later
 Route::get('/upload', function () {
     return view('modules.upload');
 });
 
-Auth::routes();
 
-Route::get('/dashboard', 'DashboardController@index')->name('dashboard');
+Auth::routes(); // includes routes for login, register, forget password
+
+// only authenticated user can access this route group
+Route::group(['prefix'=>'dashboard', 'middleware' => 'auth'], function () {
```

Comment:
group together because everything is at one place now. easy to change, easy to see which route have what permission. controller specific ,those one time middleware can put inside controller. but common one just place them in one place, easy to debug.